File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.py
import sys
input = sys.stdin.readline

tc= int(input())

for _ in range(tc):
    arr = list(input().strip())
    cnt = 0
    sumv = 0
    for v in arr:
        if v == "O":
            cnt += 1
            sumv += cnt
        else:
            cnt = 0
        
    print(sumv)
            
    

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.js
const fs = require('fs');
const stdin = fs.readFileSync('dev/stdin').toString().trim();

const ARR = stdin.split("\n").slice(1).map(s => s.split(""))

for (let arr of ARR) {
    let cnt = 0;
    let sumv = 0;
    for(let v of arr){
        if(v === "O") {
            cnt += 1;
            sumv += cnt;
        } else {
            cnt = 0;
        }
    }
    console.log(sumv);
}